client/my-sites/plan-features/actions.jsx

```diff
@@ -21,33 +21,43 @@ import { getPlanClass, isMonthly } from 'lib/plans/constants';
 import { recordTracksEvent } from 'state/analytics/actions';
 
 const PlanFeaturesActions = ( {
+	available = true,
 	canPurchase,
 	className,
-	available = true,
+	currentSitePlan,
 	current = false,
-	primaryUpgrade = false,
 	freePlan = false,
-	onUpgradeClick = noop,
+	manageHref,
+	isLandingPage,
 	isPlaceholder = false,
 	isPopular,
 	isInSignup,
-	translate,
-	manageHref,
-	isLandingPage,
+	onUpgradeClick = noop,
 	planName,
-	currentSitePlan,
 	planType,
+	primaryUpgrade = false,
+	selectedPlan,
 	recordTracksEvent: trackTracksEvent,
+	translate,
 } ) => {
 	let upgradeButton;
-	const classes = classNames(
-		'plan-features__actions-button',
-		{
-			'is-current': current,
-			'is-primary': ( primaryUpgrade && ! isPlaceholder ) || isPopular,
-		},
-		className
-	);
+	const classes = selectedPlan
+		? classNames(
```

Comment:
We have quite a bit of repetition here with this ternary. The `classNames` module allows you to provide logic to each of the classes you provide, so it would probably be best to provide the `selectedPlan` logic in there.